modules/article-page/index.js

```diff
@@ -2,7 +2,8 @@ module.exports = {
   extend: '@apostrophecms/piece-page-type',
   options: {
     label: 'Article Index Page',
-    pluralLabel: 'Article Index Pages'
+    pluralLabel: 'Article Index Pages',
+    perPage: 3
```

Comment:
Did you mean to pin this at 3 or was that just testing?

Reply:
That was for testing. I'll remove that.